src/containers/ProfileContainer.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import styled from 'styled-components'
+import { useAppDispatch, useAppSelector } from '../hooks'
+import { setClient } from '../state/clientSlice'
+
+const Container = styled.div`
+  width: 50vw;
+  height: 8rem;
+  max-width: 30rem;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin-top: 3vh;
+  z-index: 1;
```

Comment:
```suggestion
  width: 50vw;
  height: 8rem;
  max-width: 30rem;

  display: flex;
  flex-direction: row;
  justify-content: center;
  align-items: center;

  margin-top: 3vh;
  z-index: 1;
```
관련성 있는 CSS 코드를 조금 분리해보는건 어떨까요? 가독성에 도움이 될것 같습니다.

Reply:
감사합니다. 안그래도 분리하신 부분들 보고 따라해야겠다고 생각하던 차였어요!